core/block_svg.ts

```diff
@@ -1720,4 +1720,20 @@ export class BlockSvg
     traverseJson(json as unknown as {[key: string]: unknown});
     return [json];
   }
+
+  override jsonInit(json: AnyDuringMigration): void {
+    super.jsonInit(json);
+      
+    if (json['classes']) {
+      let classesToAdd = '';
+
+      if (Array.isArray(json['classes'])) {        
+        classesToAdd = json['classes'].join(' ');
+      } else {        
+        classesToAdd = json['classes'];
+      }
+      
+      this.addClass(classesToAdd);
```

Comment:
I think you can simplify this using a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator).
```suggestion
      this.addClass(Array.isArray(json['classes']) ? json['classes'].join(' ') : json['classes']);
```

You'll probably need to run `npm run format` in the root directory after applying this change to handle the line wrapping.

Reply:
I agree, thanks for pointing that out! Updated that and formatted using `npm run format`.